server/services/licenceService.js

```diff
@@ -106,6 +107,18 @@ module.exports = function createLicenceService(licenceClient, establishmentsClie
         }
     }
 
+    async function updateBassReferral(data = {}) {
+        try {
+            const nomisId = data.nomisId;
+            const bassReferralData = createBassReferralObject(data);
+
+            return await licenceClient.updateSection('bassReferral', nomisId, bassReferralData);
```

Comment:
you actually don't need to do
`return await licenceClient.updateSection('bassReferral', nomisId, bassReferralData);`
You can just do
`return licenceClient.updateSection('bassReferral', nomisId, bassReferralData);`

Because the calling function is the one what will be awaiting the response. `licenceClient.updateSection` returns a promise so does `await licenceClient.updateSection`